lldb/include/lldb/Expression/DWARFExpression.h

```diff
@@ -18,7 +18,9 @@
 #include "llvm/DebugInfo/DWARF/DWARFLocationExpression.h"
 #include <functional>
 
+namespace lldb_plugin::dwarf {
```

Comment:
Do we do this anywhere else? Should this be split into:
```
namespace lldb_plugin {
namespace dwarf {
```
Not sure if there is a llvm coding convention for this?

Reply:
This is possible after the switch to c++17, and that's probably why no one else is using it in LLDB. There's no explicit llvm coding convention for this. I'm fine with splitting it into two lines if you want.

Reply:
I'd prefer the old way for the sake of consistency.

Reply:
Sure, I'll change that!!